### PR TITLE
[Event] Change equals from comapring all fields to compare by ID

### PR DESCRIPTION
**Pull request by makrushinoff**

### CODE REVIEW

File: src/test/java/ua/kpi/iasa/IASA_Organiser/controller/ControllerTest.java [hunk 1]
```diff
@@ -97,7 +97,7 @@ public void shouldCreateNewEvent() {
     public void shouldChangeEvent() {
         controller.setEventService(eventService);
 
-        controller.changeEvent(event);
+        controller.updateEvent(event);
```
Comment (VasiliyPetrov1) [inline src/test/java/ua/kpi/iasa/IASA_Organiser/controller/ControllerTest.java:100]: Please make git fetch

Comment (VasiliyPetrov1) [reply src/test/java/ua/kpi/iasa/IASA_Organiser/controller/ControllerTest.java:100]: Fix it and call me in Telegram group when you done that task

Comment (makrushinoff) [reply src/test/java/ua/kpi/iasa/IASA_Organiser/controller/ControllerTest.java]: Made it)

**Review by VasiliyPetrov1 — COMMENTED** [review]
All is ok

**Review by MLGMAG — COMMENTED** [review]
All right!

**Review by MLGMAG — COMMENTED** [review]
All cool

**Review by MLGMAG — APPROVED** [review]

**Review by VasiliyPetrov1 — APPROVED** [review]

**Pull request merged**